innersource-portal.md

```diff
@@ -0,0 +1,48 @@
+## Title
+
+* InnerSource Portal
+
+## Patlet
```

Comment:
I propose to shorten the patlet. If I remember Bob right, the rule of thumb is to limit a patlet to two sentences.

Reply:
Got it.  Will do.